ForgeRock Cloud APIs.postman_collection.json

```diff
@@ -4258,6 +4269,526 @@
+										"header": [
+											{
+												"key": "Authorization",
+												"value": "Bearer {{accessToken}}"
+											},
+											{
+												"key": "Content-Type",
+												"name": "Content-Type",
+												"type": "text",
+												"value": "application/json"
+											}
+										],
+										"body": {
+											"mode": "raw",
+											"raw": ""
+										},
+										"url": {
+											"raw": "{{tenantApiV1Url}}/hosted-pages",
+											"host": [
+												"{{tenantApiV1Url}}"
+											],
+											"path": [
+												"hosted-pages"
+											]
+										}
+									},
+									"status": "OK",
+									"code": 200,
+									"_postman_previewlanguage": "json",
+									"header": [
+										{
+											"key": "X-Powered-By",
+											"value": "Express"
+										},
+										{
+											"key": "Access-Control-Allow-Origin",
+											"value": "*"
+										},
+										{
+											"key": "Content-Type",
+											"value": "application/json; charset=utf-8"
+										},
+										{
+											"key": "Content-Length",
+											"value": "307"
+										},
+										{
+											"key": "ETag",
+											"value": "W/\"133-YlpGUJfUlWemDBqNwzG2KZRxqmA\""
+										},
+										{
+											"key": "Date",
+											"value": "Thu, 28 Mar 2019 22:33:31 GMT"
+										},
+										{
+											"key": "Via",
+											"value": "1.1 google"
+										}
+									],
+									"cookie": [],
+									"body": "{\n    \"cssUrl\": \"https://ui-docs51.forgeblocks.com/css/hosted.css\",\n    \"forgotPasswordTitle\": \"Reset Password\",\n    \"logoUrl\": \"https://ui-docs51.forgeblocks.com/img/logo-forgerock.png\",\n    \"recoverUsernameTitle\": \"Recover Your Username\",\n    \"resetPasswordTitle\": \"Reset Your Password\",\n    \"signInTitle\": \"Sign In\",\n    \"signUpTitle\": \"Sign Up\"\n}"
+								}
+							]
+						},
+						{
+							"name": "Modify Existing Hosted Page",
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "{{accessToken}}",
+											"type": "string"
+										}
+									]
+								},
+								"method": "PUT",
+								"header": [
+									{
+										"key": "Content-Type",
+										"name": "Content-Type",
+										"type": "text",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"cssUrl\": \"https://ui-docs51.forgeblocks.com/css/hosted.css\",\n    \"forgotPasswordTitle\": \"Reset Your Password\",\n    \"logoUrl\": \"https://ui-docs51.forgeblocks.com/img/logo-forgerock.png\",\n    \"recoverUsernameTitle\": \"Recover Your Username\",\n    \"resetPasswordTitle\": \"Reset Password\",\n    \"signInTitle\": \"Sign In\",\n    \"signUpTitle\": \"Sign Up\"\n}"
+								},
+								"url": {
+									"raw": "{{tenantApiV1Url}}/hosted-pages",
```

Comment:
@chuckmasucci I see this is slightly different (plural) than what's shown under the API Scopes tab (`/hosted-page`)

Reply:
The plural is incorrect. The API endpoint is `hosted-page`.

Reply:
Chuck, the singular endpoint doesn't work for me, at least for a staging environment created late last week (March 28).

And if I understand correctly, it's [hosted-pages in the repo](https://github.com/ForgeCloud/saas/blob/09f71b7b0710989db83307f80d522d05fd6ff475/services/org-ui/src/client/private/services/hosted-pages.ts),

Reply:
@chuckmasucci  has confirmed via DM that it's /hosted-pages